pkg/operator/minio/controller.go

```diff
@@ -322,7 +322,7 @@ func (c *Controller) onAdd(obj interface{}) {
 	// Validate object store config.
 	err := validateObjectStoreSpec(objectstore.Spec)
 	if err != nil {
-		logger.Errorf("failed to validate object store config")
+		logger.Errorf("failed to validate object store config: %s", err)
```

Comment:
Change the `:` to `.` as in the other errors returned (e.g., line 306, 302) and use `%+v` instead of `%s`.

Reply:
fixed